src/components/Requirements/RequirementSideBar.vue

```diff
@@ -244,15 +252,17 @@ export default defineComponent({
     CopyPlanModal,
     NamePlanModal,
     EditPlanModal,
+    CollectionsSideBar,
   },
   props: {
     startTour: { type: Boolean, required: true },
     isDisplayingMobile: { type: Boolean, required: true },
     isMobile: { type: Boolean, required: true },
     isMinimized: { type: Boolean, required: true },
     startNewFeatureTour: { type: Boolean, required: true },
+    isDisplayingCollection: { type: Boolean, required: true },
   },
-  emits: ['showTourEndWindow', 'toggleMinimized'],
+  emits: ['showTourEndWindow', 'toggleMinimized'], // probably will remove emits with just using components
```

Comment:
Could you clarify what you meant here?

Reply:
Sorry that's an old comment. Originally, I made an 'emit' to open the sidebar, then completely scratching that idea.